akvo/rsr/spa/app/modules/editor/main-menu.jsx

```diff
@@ -12,7 +12,7 @@ const keyDict = {
   settings: 'Validation set',
   info: 'General information',
   contacts: 'Project Contacts',
-  partners: 'Partners',
+  partners: 'Partners & User Access',
```

Comment:
Why edit this?!

Reply:
Because then we can remove this line?

`if (key === 'partners' && !shouldShowFlag(userRdr.organisations, flagOrgs.DISABLE_FAC)) return 'Partners & User Access'`

Reply:
Ah okay I see. Let me look into it